shared/desktop/webpack.config.development.js

```diff
@@ -1,45 +1,16 @@
 /* eslint-disable flowtype/require-valid-file-annotation */
-const path = require('path')
 const webpack = require('webpack')
-const webpackTargetElectronRenderer = require('webpack-target-electron-renderer')
 const baseConfig = require('./webpack.config.base')
 const getenv = require('getenv')
 const UnusedFilesWebpackPlugin = require('unused-files-webpack-plugin').default
 const DashboardPlugin = require('webpack-dashboard/plugin')
+const {isHot, HMRUrl, RHLPatch, noSourceMaps, mockRule} = require('./webpack.common')
 
-const USING_DLL = getenv.boolish('USING_DLL', false)
-const NO_SERVER = getenv.boolish('NO_SERVER', false)
-const NO_SOURCE_MAPS = getenv.boolish('NO_SOURCE_MAPS', false)
-const HOT = getenv.boolish('HOT', false)
-const defines = {
-  __DEV__: true,
-  __SCREENSHOT__: false,
-  'process.env.NODE_ENV': JSON.stringify('development'),
-  __VERSION__: JSON.stringify('Development'),
-}
-
-const config = Object.assign({}, baseConfig)
```

Comment:
instead of doing a shallow copy and then editing the innards we use rest spread and build our own config